indico/modules/events/editing/schemas.py

```diff
@@ -246,11 +246,15 @@ class Meta(EditableSchema.Meta):
 class EditableBasicSchema(mm.SQLAlchemyAutoSchema):
     class Meta:
         model = Editable
-        fields = ('id', 'type', 'state', 'editor', 'timeline_url', 'revision_count')
+        fields = ('id', 'type', 'state', 'editor', 'timeline_url', 'revision_count', 'tags')
 
     state = EnumField(EditableState)
     editor = fields.Nested(EditingUserSchema)
     timeline_url = fields.String()
+    tags = fields.Method('_get_tags')
+
+    def _get_tags(self, editable):
+        return [EditingTagSchema().dump(t) for t in editable.latest_revision.tags]
```

Comment:
You can use `many=True` ;)
```suggestion
        return EditingTagSchema(many=True).dump(editable.latest_revision.tags)
```

or even get rid of the method entirely:

```python
tags = fields.List(fields.Nested(EditingTagSchema), attribute='latest_revision.tags')
```